### PR TITLE
initialise Trainer Notes page

### PR DESCRIPTION
**Pull request by tobyhodges**
Started building up the page of trainer notes

### CODE REVIEW

File: instructors/instructor-notes.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 ---
 
-### Training Themes
+## Schedule
 
-#### Lesson Development as a Learning Process
+The curriculum was piloted in six half-days, 
+with an extended break for lesson trial runs between half-days 4 & 5.
+Below is the schedule for the pilot, showing a break down of episodes
+across each half-day.
```
Comment (sstevens2) [inline instructors/instructor-notes.md:10]: I couldn't find the schedule mentioned below.  Do we still need to add a table or something with the schedule?

That said, do we want to maintain this manually if we rearrange the episodes for each day?

**Review by sstevens2 — CHANGES_REQUESTED** [review]
Not quite sure where the schedule mention is.  Should it be a table that shows which episodes are on each half day? Small concern about remembering to update the table if we rearrange anything.

**Comment by tobyhodges** [comment]
Good catch @sstevens2. You are right that it is too early to provide a schedule now, and I have removed the sentence in 1fac2be

I think it may be helpful to include such a schedule at some point when the order and distribution of episodes has settled down a bit more. For example, it would enable us to remove the break .md files in the lesson site, instead pointing Trainers to this page (or somewhere else) for guidance on how to split the content over multiple days.

**Review by sstevens2 — APPROVED** [review]
That makes sense @tobyhodges.  I'll get this merged.

**Pull request merged**